web-console/project-details/add-to-project.md

```diff
@@ -8,16 +8,13 @@
 ![template](img/QuickSearch.png)
 - A search box will be added to the secondary masthead to the left of the "Quick Add" dropdown.
 - This search feature will behave like [Search](https://github.com/openshift/openshift-origin-design/blob/master/web-console/homepage/search.m) on the landing page.
-- Selecting an item from the search will initiate the ordering flow on the [Overlay Panel](http://openshift.github.io/openshift-origin-design/web-console/patterns/overlay-panel) on the projects detail page.
+- Selecting an item from the search will initiate the ordering flow on the [Overlay Panel](http://openshift.github.io/openshift-origin-design/web-console/homepage/search-filter#search-catalog) on the projects detail page.
 - The Project selection step should be hidden from the ordering panel in this case since the user is adding from within the console and we know what Project.
 
 ### Browse Catalog
-![project browse](img/project-browse-01.png)
-- Selecting `Browse Catalog` from the dropdown or clicking the link to view more search results will bring up a version of the catalog in a modal.
-- A filtered view of the catalog will be brought up if the quick search bar is used to trigger the catalog.
-- Selecting a catalog item should replace the browse modal with the ordering overlay.
-- Clicking the X button, or the `Return to Project` link will close the modal and take the user back to where they were when they opened the catalog.
-- Closing the ordering overlay should cancel the `Browse Catalog` action.
+- Selecting `Browse Catalog` from the dropdown or clicking the link to view more search results will navigate the user to the [Catlog](http://openshift.github.io/openshift-origin-design/web-console/projject-details/in-project-catalog) inside that project.
```

Comment:
@ncameronbritt lil typo "Catlog"

Reply:
I misspelled "catalog" literally every single time I typed it!

Reply:
Also typo: `projject-details`